Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -283,11 +283,19 @@ class PlaygroundController: ObservableObject {
     var customerSessionClientSecret: String?
     var paymentMethodTypes: [String]?
     var amount: Int?
-    var checkoutEndpoint: String = PaymentSheetTestPlaygroundSettings.defaultCheckoutEndpoint
     var addressViewController: AddressViewController?
     var appearance = PaymentSheet.Appearance.default
     var currentDataTask: URLSessionDataTask?
 
+    var checkoutEndpoint: String {
+        get {
+            settings.checkoutEndpoint
```

Comment:
This still has the same default value as before, which is set here: https://github.com/stripe/stripe-ios/blob/dce44782e444f0d8d88df9dc2bc17676caa22fe7/Example/PaymentSheet%20Example/PaymentSheet%20Example/PaymentSheetTestPlaygroundSettings.swift#L473